src/runtime/runtime.js

```diff
@@ -161,8 +161,12 @@ export class Runtime {
     /** @private {?PageConfigResolver} */
     this.pageConfigResolver_ = null;
 
+    /** @private @const {!ClientEventManager} */
+
+    this.eventManager_ = new ClientEventManager(this.configuredPromise_);
```

Comment:
Unfortunately, we cannot create an event manager in runtime unless we also create a way to pass that event manager into configured runtime.  This is because there should only ever be 1 event manager (otherwise things might not all subscribe to the same one).  

We can create a method to pass event manager into configuredruntime, and we did have a way of doing this briefly but we went with the configuration promise instead because AMP couldn't instantiate event manager anyway.

The easiest thing you can do for your API is to pass in this.configuredPromise_.  Then, within your code:

this.configuredPromise_.then( configuredRuntime => 
  configuredRuntime.eventManager().then( em => 
    em.logSwgEvent( ... )
  )
);

Reply:
Of course, I remember this conversation now. Sorry for the mental lapse (long weekend!).